vllm/core/block/prefix_caching_block.py

```diff
@@ -507,7 +516,10 @@ def mark_blocks_as_accessed(self, block_ids: List[int],
                     "Mark block as accessed which is not belonged to GPU")
 
     def mark_blocks_as_computed(self, block_ids: List[int]) -> None:
-        raise NotImplementedError("Marking as computed is incremental")
+        # Mark all touched blocks as computed.
+        for block_id in self._touched_blocks:
+            self._block_tracker[block_id].computed = True
```

Comment:
can we try to avoid having to iterate through the entire list? if we reverse the iteration we should be able to break on the first already computed block

Reply:
All blocks in this list (or set) should be uncomputed, so use `set`  should be sufficient to de-duplicate?